wiki/Beatmap/Pattern/Mania/Chords/en.md

```diff
@@ -0,0 +1,33 @@
+---
+stub: true
+tags:
+  - chords
+---
+
+# Chords
+
+A **chord** is two or more notes that are hit at the same time. They are known to increase density and highlight different sounds within the music.
```

Comment:
```suggestion
A **chord** is a group of two or more notes that are hit at the same time. They are known to increase density and highlight different sounds within the music.
```

Should we also clarify that a chord is rather a group of notes that are snapped at the same timing point? Given that you could technically hit a chord without hitting it at the "same time"